HW1/P2/P2b.py

```diff
@@ -0,0 +1,27 @@
+from P2 import *
+
+# Your code here
+import findspark
+findspark.init()
+import pyspark
+import random
+
+sc = pyspark.SparkContext(appName="Spark2b")
+
+# make pyspark shut up
+sc.setLogLevel('WARN')
+
+imagei = sc.parallelize([x for x in range(2000)], 10)
+image = imagei.cartesian(imagei)
+
+mandelbrots = image.map(lambda (i,j): ((i, j), mandelbrot(j/500.0 - 2, i/500.0 - 2)))
+# repartition randomly
+mandelbrots_repartitioned = mandelbrots.partitionBy(100, lambda _: random.randint(0, 99))
```

Comment:
The randint range should probably be (0, 100).
It is unclear why you first compute the mandelbrot, and only then repartition, and not the other way around.